### DISCUSSION TITLE
Alguém ai com indicação ou exemplo de como usar o translate na nova versão do NextJS?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#1933] [Perguntas e Respostas] [answered]
Vi que a nova versão tem muita coisa legal, inclusive a parte de linguagem built-in, mas não vi exemplos de como fazer o translate, apenas como pegar a linguagem... Alguém ai tem algum exemplo?

Muito obrigado, abraços!

**Comment by willianjusten** [comment, marked as answer]
Opa, já chegou a dar uma olhada no [Next Translate](https://github.com/vinissimus/next-translate)? É uma lib de i18n feita em cima dessa nova feature do NextJS 10 =)

**Reply by willianjusten** [reply]
Tem também um [outro exemplo](https://medium.com/javascript-in-plain-english/exploring-internationalisation-with-nextjs-10-and-react-intl-b6b8ccac36c9) utilizando essa feature com o [React-intl](https://formatjs.io/docs/react-intl/)

**Reply by wilsonneto-swe** [reply]
Oopa Willian, valeu demais pela resposta.

O primeiro é o que eu uso atualmente, e me incomoda para usar ele com SSR ele desabilita o static optimization, e eu preciso criar um server customizado, acredito que as novas funcionalidades do next 10 já não precisaria de toodas estas perdas e nem do servidor customizado que pode vir a ser um problema no futuro rs

Já o segundo link que mandou, muito obrigado, estou lendo e fazendo umas POCs com ele

**Comment by Cleysonlb** [comment]
Falaaa @wilsonneto-dev, tive uma experiencia positiva com o [nexti18next](https://github.com/isaachinman/next-i18next), vale a pena dar uma olhada, ele usa o i18next como base, possibilitando chamar o `t` via context/hook, o que facilita muito evitando prophell xD

**Reply by willianjusten** [reply]
Gosto demais do nextI18next também, inclusive é o que mais uso. Mas ele não tem muita coisa diretamente ligada a Feature do NextJs 10 de locale. 

Não que não funcione bem tb ^^
